src/client/src/components/GraphProps.tsx

```diff
@@ -69,20 +69,20 @@ const updateNodes = async (
     elements: Elements,
     setElements: React.Dispatch<React.SetStateAction<Elements>>
 ): Promise<void> => {
-    for (const el of elements) {
-        if (isNode(el)) {
-            const node: INode = el.data;
-
-            if (node) {
+    const nodes: INode[] = elements
+        .map((el) => {
+            if (isNode(el) && el.data) {
+                const node = el.data;
                 node.x = el.position.x;
                 node.y = el.position.y;
-
-                await updateNode(node);
-            } else {
-                toast('❌ What is going on?');
```

Comment:
If the toast is removed, the import for it should be removed too, at least to make the linter happy